module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -660,8 +660,27 @@ protected function toHTML($data, $group = null)
                 '<a id="$1" href="#$2"',
                 $data
             );
-
             $data = $this->replaceBRWithCommas($data, $group);
+
+            // ensure tables & mathML are not shown as raw html
+            if (strpos($data, 'ephtml') > -1) {
+                $pattern = "/<\/?ephtml>/";
```

Comment:
One point of confusion -- you appear to be splitting using `<ephtml>` or `</ephtml>`, but doesn't the data actually contain `&lt;ephtml&gt;` or `&lt;/ephtml&gt;` at this point? I realize the test is passing, so I must be overlooking/misunderstanding something -- but I'm not sure I understand why we're using unencoded HTML in the regular expression but encoded HTML in the strpos statements below. If nothing else, maybe we need a comment to clarify this so I don't get confused by it again in the future. :-)

Reply:
I'll need to create a proper comment then.
Basically the data looks like this at this point:

`<epthml>&lt;math&gt;LOTS OF COMPLEX STUFF&lt;/math&gt;</ephtml>`

So we are splitting at ephtml, then URL decode, then handle from there...

Reply:
Ahh, I understand -- I got confused because looking at the JSON fixture, I saw that everything was encoded... but now I see that some things are DOUBLE encoded. That's where I went wrong. :-)